sfcasts/install.md

```diff
@@ -1,61 +1,41 @@
-goodies. I've already done all of these steps except for two, the first find your
-terminal. The first is to compile the assets. So 
-
-```terminal
+Hey friends! We are in for a treat with this tutorial! It's EasyAdmin, my favorite admin generated for Symfony. It just gives you so many features out of the box and it looks great. This shouldn't really be a surprise because its creator is the always-impressive Javier Aguila. So let's have some fun and learn how to bend EasyAdmin to our will, because getting a lot of features for free is great, as long as we can extend everything when we need to. To get the most “easy” out of  EasyAdmin, you should definitely code along with me. 
+
+You probably know the drill: Download the course code from this page and unzip it to find a start directory with the same code that you see here. Check out the `README.md` file for all these setup goodies. I've already done all of these steps except for two. 
+
+Find your terminal and run:
+
+```terminal 
 yarn install
 ```
 
-I actually did run
-that already to save time <affirmative> and then I'll actually compile my assets by
-running
+I ran this already to save time, so I'll actually compile my assets by
+running:
 
-```terminal
+```terminal 
 yarn watch
 ```
 
-You can also run `yarn dev-server`, which is another cool way to do this. Perfect. Then
-I'm gonna open up another tab and say, 
+You can also run `yarn dev-server`, which is another cool way to do this.
+
+Perfect! Then I'm going to open up another tab and say: 
 
-```terminal
+```terminal 
 symfony serve -d
 ```
 
-and that's gonna start a
-local web server using the Symfony binary and it starts at `127.0.0.1:8000`. I'm
-gonna be lazy hold command and click that to say hello to caldron overflow. You've
-been doing our Symfony five series. You're probably familiar with this project, but
-this is not a Symfony 5 project. This is a Symfony 6 project. Ooh, but don't
-worry if you're using Symfony 5, it's not gonna be, it won't be any different.
-
-Now most of the code inside of this project, you don't really need to worry about.
-The most important thing is probably our `src/Entity/` directory. Our doctrine
-entities, our site has questions. Each question has an answer, has a number of
-answers. Each question belongs to a single topic and then we have a user entity. Our
-goal in this tutorial is to create a rich admin section that allows our admin users
-to manage these. So let's get EasyAdmin installed, head terminal and run 
-
-```terminal
+That's going to start a local web server using the Symfony binary and it starts at `127.0.0.1:8000`. I'm gonna be lazy, hold ‘command’ and click that to say “hello” to Cauldron Overflow. If you've been doing our Symfony five series, you're probably familiar with this project, but this is not a Symfony 5 project. This is a Symfony 6 project. But don't worry if you're using Symfony 5. It won't be any different.
```

Comment:
GREAT example of you stopping my rambling phrasing ("and that's gonna start a local") and instead using a new sentence ❤️

Reply:
First sentence I used "start" twice - that's a good thing to try to notice and edit out.

For the URL - `127.0.0.1:8000` - you can put `https://127.0.0.1:8000`. That's tricky: I don't actually say the `https://`, but it IS part of the URL. In some rare situations like this, it's ok if the script doesn't exactly match what I say (I'll actually say when I record simple "127.0.0.1:8000" without the `https`, but it's nice to have in the script).

Reply:
"Command" => "Cmd" - referring to the key on my Mac :)

Reply:
I use a lot of ellipsis for timing. I can always add those later, but if you can "feel" some, that's great.

For example,:

> and click that to say... hello to Cauldron overlow!

The pause is because it takes a second or two for the page to load.

Reply:
five -> 5, it's a version number, so the number is better.

Reply:
Two "buts" close together in the last 2 sentences, sounds funny. Again, that's ME doing it, but it needs to be better before recording :)

Reply:
> "Command" => "Cmd" - referring to the key on my Mac :)

My Mac keyboard actually spells it out. :D I actually thought about that when I typed it and had to actually look at my key to see what would be appropriate. I'll shorten it from now on.

Reply:
> I use a lot of ellipsis for timing. I can always add those later, but if you can "feel" some, that's great.
> 
> For example,:
> 
> > and click that to say... hello to Cauldron overlow!
> 
> The pause is because it takes a second or two for the page to load.

I actually typed out "..." in almost that exact place but then I removed it because I wasn't sure if you'd want it in there or not. Noted :D

Reply:
> My Mac keyboard actually spells it out. :D I actually thought about that when I typed it and had to actually look at my key to see what would be appropriate. I'll shorten it from now on

OMG, you're right! No, you can keep `Command` - I just was totally wrong!